working-group-abilities.md

```diff
@@ -0,0 +1,37 @@
+# Working Group Abilities
+
+Established working groups in the OSSF are given broad latitude to accomplish their goals.
+The TAC aims to only require formal approval in limited cases.
+This document outlines some of the explicitly permitted activities and resources made
+available to working groups.
+It is not intended to be exhaustive, and will grow over time.
+
+When in doubt, reach out to the TAC with any questions!
+
+## Repositories
+
+We expect that working groups will need to create repositories for code, tools, and other projects.
+These should be created in the [ossf](github.com/ossf) organization where possible to simplify
+management, but working groups can also create new organizations where necessary.
+
+If any of these projects grow large enough, they can be "spun-off" into separate working groups by
+the TAC.
+As a general rule-of-thumb, if a sub-project requires its own standing meeting, it might be time to
+spin-it-off.
+
+## Papers
+
+As the working groups are intended to be the subject matter experts in their domains,
+they should author papers for release through the OpenSSF with no formal review or approval.
+However, the TAC does ask that drafts be shared as early as possible for feedback and awareness.
+
+All papers should still follow open source best practices including transparency in development and avoiding vendor bias.
+
+## Technical Infrastructure
+
+For the time being, the OSSF has limited formal resources (cloud credits, API keys, etc.) available for
+hosting or project development.
+Working groups are free to use whatever resources they can find, or to solicit help from member
+organizations.
+The TAC asks that working groups keep them informed of what infrastructure they are using for awareness
+as we begin the process of collecting formal resources.
```

Comment:
The OpenSSF does now have funding.  Let's switch up this section to describe how to request resources.

Reply:
Any suggestions? Should I just drop this now? I don't think we have that process and don't want to hold up the rest of this on that detail.

Reply:
I'm OK with dropping.

Reply:
Reworded a bit.